scripts/rust-utils/README.md

```diff
@@ -0,0 +1,34 @@
+# Rust Utils Workspace
```

Comment:
Move this information to the wiki of this repository (that way everything is in one place) :)